hare/builder.go

```diff
@@ -10,17 +10,19 @@ import (
 	"github.com/spacemeshos/go-spacemesh/signing"
 )
 
+//go:generate scalegen
+
 // Message is the tuple of a message and its corresponding signature.
 type Message struct {
 	Sig      []byte
-	InnerMsg *innerMessage
+	InnerMsg *InnerMessage
```

Comment:
i think this should not be an option

Reply:
if I make it a struct rather than a pointer then how would I check `InnerMessage` is not empty? I mean what field shall I look at?

Reply:
what do you mean? it can't be nil if it won't be a pointer